### PR TITLE
Add yelp_xsl

### PR DESCRIPTION
**Pull request by lovedheart**

### CODE REVIEW

File: packages/yelp_xsl.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+require 'package'
+
+class Yelp_xsl < Package
+  description 'yelp-xsl is a collection of programs and data files to help you build, maintain, and distribute documen$
```
Comment (uberhacker) [inline packages/yelp_xsl.rb]: It looks like you missed the last part of the description and there is no closing quote at the end.

File: packages/yelp_xsl.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+require 'package'
+
+class Yelp_xsl < Package
+  description 'yelp-xsl is a collection of programs and data files to help you build, maintain, and distribute documen'
```
Comment (uberhacker) [inline packages/yelp_xsl.rb]: Still missing the last part of the description.  Should it be `documentation`?

Comment (uberhacker) [reply packages/yelp_xsl.rb]: Please slow down and pay attention to detail. :)

Comment (lovedheart) [reply packages/yelp_xsl.rb]: I have tested it several times.

**Pull request merged**